webrtc.html

```diff
@@ -9495,7 +9495,16 @@ <h3>Introduction</h3>
       queries can be linked by the <code><a>RTCStats</a></code> <a data-for=
       "RTCStats">id</a> dictionary member. Thus, a Web application can make
       measurements over a given time period by requesting measurements at the
-      beginning and end of that period.</p>
+        beginning and end of that period.</p>
+      <p>
+        Stats objects may have a limited lifetime. Until the end of
+        their lifetime, they are always present in the result from
+        getStats(). When their lifetime ends, a record of the
+        statistics for that object is emitted through a
+        "statslifetimeended" event, containing an RTCStats
+        dictionary. The object descriptions in [[WEBRTC-STATS]]
+        describe the lifetime of each stats object type.
```

Comment:
Should we add some guidance here? I know we discussed quite conservative definitions of lifetime termination that left out all terminal cases like `transceiver.stop()` and `pc.close()`. The purpose of these events seem limited to distinguishing old data from new data in subsequent getStats() calls, e.g. replaced tracks, and restarted ice candidates.

Reply:
Yes, there should be guidance.

Reply:
I think that belongs in the "stats object design guidelines" in the stats document.